fineract-e2e-tests-runner/src/test/resources/features/LoanDelinquency.feature

```diff
@@ -625,10 +625,7 @@ Feature: LoanDelinquency
       | PAUSE  | 15 October 2023 | 30 October 2023 |
     Then Loan has the following LOAN level delinquency data:
       | classification | delinquentAmount | delinquentDate  | delinquentDays | pastDueDays |
-      | RANGE_3        | 250.0            | 04 October 2023 | 11             | 14          |
-    Then Loan has the following INSTALLMENT level delinquency data:
-      | rangeId | Range   | Amount |
-      | 2       | RANGE_3 | 250.00 |
+      | NO_DELINQUENCY | 250.0            | 04 October 2023 | 11             | 14          |
```

Comment:
I think these tests should be fixed to run the delinquency business step properly and not by changing the outcome. Would you consider discuss these things with @peter-kovacs-dpc ?